### ISSUE TITLE
[regression] search_module_defaults() missing 2 required positional arguments

### ISSUE THREAD

**Issue by alexandrevicenzi** [release-engineering/ubi-population-tool#64]
ubipop command is broken, integration test is failing.

```python
    def run(self):
        if not self.future.set_running_or_notify_cancel():
            return
    
        try:
>           result = self.fn(*self.args, **self.kwargs)
E           TypeError: search_module_defaults() missing 2 required positional arguments: 'name' and 'stream'
```

**Comment by alexandrevicenzi** [comment]
Fix is in #63.

**Comment by alexandrevicenzi** [comment]
#63 was merged.
